typescript/my-widget-service/widget_service.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// This file is licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+// http://aws.amazon.com/apache2.0/
+//
+// This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+import cdk = require('@aws-cdk/cdk');
+import apigateway = require('@aws-cdk/aws-apigateway');
+import lambda = require('@aws-cdk/aws-lambda');
+import s3 = require('@aws-cdk/aws-s3');
+
+export class WidgetService extends cdk.Construct {
+  constructor(scope: cdk.Construct, id: string) {
+    super(scope, id);
+
+    const bucket = new s3.Bucket(this, 'WidgetStore');
+
+    const handler = new lambda.Function(this, 'WidgetHandler', {
+      runtime: lambda.Runtime.NodeJS810,  // So we can use async in widget.js
+      code: lambda.Code.directory('.'),
```

Comment:
This will publish the entire CDK app directory as a Lambda.

Don't you want the actual Lambda source in a subdirectory?